clab/multicluster/kind.clab.yml

```diff
@@ -116,3 +119,7 @@ topology:
     - endpoints: ["pe-kind-b-control-plane:toswitch", "leafkind-sw-b:kindctrlplb"]
     - endpoints: ["pe-kind-a-worker:toswitch", "leafkind-sw-a:kindworkera"]
     - endpoints: ["pe-kind-b-worker:toswitch", "leafkind-sw-b:kindworkerb"]
+    - endpoints: ["pe-kind-a-control-plane:migration", "migration-net:pe-kind-a-ctrl"]
```

Comment:
let's give this a different name rather than "migration". Migration makes sense only in the context of the example